posts/2021/04/spot-the-differences.md

```diff
@@ -0,0 +1,213 @@
+on our Github issues?
+
+**[Isabela]** First things first, it isn’t actually witchcraft even if it 
+looks magical from the outside. How I work depends somewhat on what problem 
+e are trying to solve, so let’s use the design of [custom icons](https://github.com/spyder-ide/ux-improvements/issues/33#issuecomment-776376943) for Spyder 5 
+as an example. Once I had a defined list of icons to work on, I needed to 
+spend time making progress on my own. Research on best practices for the 
+relevant area of design is where I started; in this case, I knew we were 
+going to be working with [Material Design Icons’](https://materialdesignicons.com/) specifications. After that, 
+I did a combination of pen-and-paper sketching and working digitally based on 
+the existing icons in Spyder and Material Design Icons while I kept note of 
+the pros and cons for different directions. I also collected design elements 
+as I built them so that I could make more consistent, accurate designs faster 
+as I worked. For the icons, this included things like letters, rounded 
+corners, and templates for the size and spacing of elements. Finally, I 
+compared options side by side and tried them out in the interface to evaluate 
+what designs were strong enough to bring to the rest of the team. Then we 
+discussed the options together.
+
+![Spyder 5 icon mockups](/images/spyder5-wipicons.png)
+
+### 6. Mockups Vs Reality
+
+**[Juanita]** After many discussions, mock-ups, and meetings, decisions had 
+been made. It was time to implement! A big part of the improvements were made 
+in [QDarkStyleSheet](https://github.com/ColinDuquesnoy/QDarkStyleSheet/) where we did the new palette and color system for both the 
+dark and light themes of Spyder. In my opinion, this was the hardest part of 
+the process since it involved getting familiar with the code first and then, 
+trying and trying again changing lines of code to change the color or style 
+of buttons, tabs, toolbars, borders, etc… 
+
+The other problem that I ran into, was trying to meet the designs’ 
+specifications. Specially, when working with the [toolbars](https://github.com/spyder-ide/ux-improvements/issues/28), figuring the right 
+number for the pixels of margins and sizes was a challenge. I tried several 
+values before finding one that closely matched the proposed mock-up only to 
+realize later that “pixels” was not the best unit for the specifications. I 
+ended up using “em” since it was more consistent across operating systems.
+Isabela, Stephanie and Carlos were part of this process as well. Between the 
+four of us we managed to implement all the changes that we had planned for 
+Spyder 5, the new color palette, the complete redesign of toolbars and the 
+new set of icons. It was an arduous task, more than we all expected, but at 
+the end we were all very happy with the results and thankful to Isabela for 
+helping us to give a new face to Spyder. 
+
+## What did we get?
```

Comment:
Maybe: The final result


Like a reveal

Reply:
Not quite sure about this one since we are trying to make all the section headers be questions. What do you think @isabela-pf?

Reply:
I do think it might stand out to have only one header in this section not be a question. Would `What's the final result?` work?